### PR TITLE
211 load test (Spike)

### LINKED ISSUE #211: Load test Oculus
We need to get a better idea on how far we can push the Oculus in terms of data points. This is to load test the hardware on the Oculus for memory. A rough idea is:
- Make a button and process to generate n random points
- Deploy to a preview site
- Load into the oculus and see when it breaks

### PR DESCRIPTION
**Pull request by Jesster2829**
Closes #211 

# What was the issue
We needed a way to load test the hardware of the oculus, so devised a way to create random datapoints and any number of them. 

# What was done and why
Outside of the VR env I created a text input and button that will show a certain amount of data point specified by the user. 

# How it was tested
Did not add units tests since this will most likely not be placed into the final project. I just did visual/manual tests such as confirming that the data points stay within the bounds of the axis, and the number of data points being correct to the number input by user.

### CODE REVIEW

**Comment by matthew-buglass** [comment]
Turns out we are even more limited than we thought. This is testing with 1k, 5k, and 10k points: https://drive.google.com/file/d/1TWzpg2WEkMuHG3R4nsWml7xDymg1FGOR/view?usp=sharing

**Comment by matthew-buglass** [comment]
I refactored the data points to be flat circles instead of spheres. It helped a little bit, but there's still a lot of overhead that we're keeping around.

https://drive.google.com/file/d/1TWQG5Q4lPQkt3k8ERDhLcDefaGDWhejd/view?usp=drivesdk

File: my-webxr-app/src/components/DataPoint.tsx [hunk 1]
```diff
@@ -38,26 +42,35 @@ export default function DataPoint({
...
+          attach="material"
+          map={CircleImg}
+          color={0x000000}
+          size={0.01}
+          sizeAttenuation
+          transparent={false}
+          alphaTest={0.5}
+          opacity={1.0}
+        />
+      </points>
```
Comment (Subzeero) [inline my-webxr-app/src/components/DataPoint.tsx:73]: The `Points` class is a great idea. Are they still interactive like this? We'll have to see what others optimizations may be possible.

Comment (matthew-buglass) [reply my-webxr-app/src/components/DataPoint.tsx:73]: Not immediately. There would be more work to do than we have time to accomplish.

**Pull request closed without merging**